crates/bevy_pbr/src/render/mesh_view_bindings.rs

```diff
@@ -33,7 +36,9 @@ use crate::{
     },
     prepass, FogMeta, GlobalLightMeta, GpuFog, GpuLights, GpuPointLights, LightMeta,
     LightProbesBuffer, LightProbesUniform, MeshPipeline, MeshPipelineKey, RenderViewLightProbes,
-    ScreenSpaceAmbientOcclusionTextures, ShadowSamplers, ViewClusterBindings, ViewShadowBindings,
+    ScreenSpaceAmbientOcclusionTextures, ScreenSpaceReflectionsBuffer,
+    ScreenSpaceReflectionsSettings, ShadowSamplers, ViewClusterBindings, ViewShadowBindings,
+    CLUSTERED_FORWARD_STORAGE_BUFFER_COUNT, MESH_PIPELINE_VIEW_LAYOUT_SAFE_MAX_TEXTURES,
```

Comment:
`CLUSTERED_FORWARD_STORAGE_BUFFER_COUNT` is only available with `debug_assertions`, you must import it on its own behind the cfg check

Reply:
I'm confused. `CLUSTERED_FORWARD_STORAGE_BUFFER_COUNT` seems to be available in `bevy_pbr/src/render/light.rs` without a `#[cfg(debug_assertions)]` above it.

Reply:
I think you meant `MESH_PIPELINE_VIEW_LAYOUT_SAFE_MAX_TEXTURES`. I fixed this.

Reply:
right sorry 🤦